### DISCUSSION TITLE
Livewire component without livewire folder.

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#7745] [1. Help]
Hello, how to create a Livewire component that will be in the resources/views/profile folder?
JetStream has components that are not in the livewire folder, but I don’t know how to create such a component.

**Comment by PhiloNL** [comment]
You can register a component manually like:

`Livewire::component('wire-elements-modal', Modal::class);`

You can change the render method of your component to choose your view:

```php
    public function render(): View
    {
        return view('wire-elements-modal::modal');
    }
```

In this example, I'm using a custom view namespace. You can learn more about this [here](https://laravel.com/docs/10.x/packages#views).
